packages/roosterjs-content-model-plugins/lib/tableEdit/editors/TableEditor.ts

```diff
@@ -145,10 +147,11 @@ export class TableEditor {
                     if (i === 0 && topOrSide == TOP_OR_SIDE.top) {
                         const center = (tdRect.left + tdRect.right) / 2;
                         const isOnRightHalf = this.isRTL ? x < center : x > center;
-                        this.setInserterTd(
-                            isOnRightHalf ? td : tr.cells[j - 1],
-                            false /*isHorizontal*/
-                        );
+                        !this.disableFeatures?.includes('VerticalTableInserter') &&
```

Comment:
nit: you can create a private function isFeatureEnabled() so no need to call .includes() everywhere